simulation_parameters/Constants.cs

```diff
@@ -51,7 +51,7 @@ public static class Constants
     // Should be the same as its counterpart in shaders/CompoundCloudPlane.shader
     public const float CLOUD_NOISE_UV_OFFSET_MULTIPLIER = 2.5f;
 
-    public const float CLOUD_CHEAT_DENSITY = 16000.0f;
+    public const float CLOUD_CHEAT_DENSITY = 1600000.0f;
```

Comment:
I think this should be put back to the previous value as I think it's a good value to test gradual cell growth. The infinite compounds cheat is way more effective at testing immediately filling up.